includes/discovery/arp-table.inc.php

```diff
@@ -4,6 +4,16 @@
 
 echo 'ARP Table : ';
 
+if( key_exists('vrf_lite_cisco', $device) && (count($device['vrf_lite_cisco'])!=0) ){
+    $vrfs_lite_cisco = $device['vrf_lite_cisco'];
+}
+else  {
+    $vrfs_lite_cisco = array(array('context_name'=>null));
+}
+
+foreach ($vrfs_lite_cisco as $vrf) {
```

Comment:
This seems to be the same as the other PR.

I'm not convinced this code does what is intended. You are setting $vrfs_lite_cisco to a string if vrf_lite_cisco is detected and then trying to loop through it. If it's not found vrf_lite_cisco then your setting an array into an array and trying to loop through it.